goblint-http-server/goblint_http.ml

```diff
@@ -6,6 +6,7 @@ open Lwt.Infix
 module Yojson_conv = Ppx_yojson_conv_lib.Yojson_conv
 
 let docroot = ref "run"
+let distroot = ref "gobview_dist"
```

Comment:
This corresponds to a subdirectory in Goblint's repo, right? Maybe this path could just be `gobview/dist` so there isn't an extra subdirectory in the Goblint repo.

Reply:
A `system` action calling `cp` maybe works? https://dune.readthedocs.io/en/stable/reference/actions.html#action-system